### DISCUSSION TITLE
Add keybind to toggle taskbar (&/or toggle taskbar auto-hide

### DISCUSSION THREAD

**Discussion by leighss** [valinet/ExplorerPatcher#2717] [Ideas]
Hi!

Love ExplorerPatcher - Thanks!

In my linux i3 (Dualboot with Win 11) I have a keybind to toggle the bar

The same here would be great - I find the auto-hide annoying as its easy to accidentally show the bar (e.g. when moving to change browser tabs)

**Comment by pyrates999** [comment]
https://www.makeuseof.com/windows-11-hide-taskbar-shortcuts/

**Reply by leighss** [reply]
Thanks!
However, ....
The turn on shortcut works for me but not the turn off one

I cant see any difference between the two code boxes for them both:

Turn on:

powershell -command "&{$p='HKCU:SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\StuckRects3';$v=(Get-ItemProperty -Path $p).Settings;$v[8]=3;&Set-ItemProperty -Path $p -Name Settings -Value $v;&Stop-Process -f -ProcessName explorer}"

Turn off:

powershell -command "&{$p='HKCU:SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\StuckRects3';$v=(Get-ItemProperty -Path $p).Settings;$v[8]=2;&Set-ItemProperty -Path $p -Name Settings -Value $v;&Stop-Process -f -ProcessName explorer}"

is (this a copy paste?) typo?

or am I missing something obvious?

**Reply by pyrates999** [reply]
you gotta figure that part out yourself

**Reply by leighss** [reply]
Oh - I just noticed they are different:
Settings;$v[8]=3 vs Settings;$v[8]=2
But the turn off one doesn't work, and I know nothing about Windows Powershell.

**Reply by leighss** [reply]
https://stackoverflow.com/questions/31416438/how-to-auto-hide-the-taskbar-from-the-command-line
